src/mesh/gmv_io.C

```diff
@@ -2247,32 +2195,20 @@ void GMVIO::copy_nodal_solution(EquationSystems & es)
       // Get a generic reference to the current System
       System & system = es.get_system(sys);
 
-      // And a reference to that system's dof_map
-      // const DofMap & dof_map = system.get_dof_map();
-
       // For each var entry in the _nodal_data map, try to find
       // that var in the system
-      std::map<std::string, std::vector<Number>>::iterator it = _nodal_data.begin();
-      const std::map<std::string, std::vector<Number>>::iterator end = _nodal_data.end();
-      for (; it != end; ++it)
+      for (const auto & pr : _nodal_data)
         {
-          std::string var_name = it->first;
-          // libMesh::out << "Searching for var " << var_name << " in system " << sys << std::endl;
+          std::string var_name = pr.first;
```

Comment:
While we're playing with the code here anyway, should this be a reference-to-const?

Reply:
Definitely, no reason to make a copy.